components/Navigation.tsx

```diff
@@ -437,6 +444,22 @@ export default function Navigation() {
         >
           Gateway
         </NavigationLink>
+        <NavigationLink
+          href="/topics/experiments"
+          subLinks={
+            <Fragment>
+              <NavigationSubLink href="/topics/experiments#fingerprints">Fingerprints</NavigationSubLink>
+              <NavigationSubLink href="/topics/experiments#rollouts">Rollouts</NavigationSubLink>
+              <NavigationSubLink href="/topics/experiments#user-experiments">User Experiments</NavigationSubLink>
+              <NavigationSubLink href="/topics/experiments#guild-experiments">Guild Experiments</NavigationSubLink>
+              <NavigationSubLink href="/topics/experiments#get-experiment-assignments">
+                Get Experiment Assignments
+              </NavigationSubLink>
```

Comment:
```suggestion
              <NavigationSubLink href="/topics/experiments#endpoints">
                Endpoints
              </NavigationSubLink>
```

Reply:
This is inconsistent with all the other pages

Reply:
Nah it's consistent with all the changed ones, only inconsistent with the ones I haven't gotten to yet